<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7SBKKJ1" failures="1" tests="8" name="Default test" time="43.868" errors="0" timestamp="2025-02-20T17:06:55 IST">
  <testcase classname="SIP.Dealer_User_Login" name="Dealer_User_Login1" time="6.579"/>
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Click_On_SIP" time="4.524"/>
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Click_On_Additional_Fillter_To_Clear_Data" time="6.451"/>
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Click_On_Customer_Page" time="0.095"/>
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Click_On_Customer_Query" time="11.532">
    <failure type="java.lang.AssertionError" message="(In-Pending Booking),Tentative Waiting period not visible as Expected Time. ">
      <![CDATA[java.lang.AssertionError: (In-Pending Booking),Tentative Waiting period not visible as Expected Time. 
at org.testng.Assert.fail(Assert.java:99)
at SIP.Select_PRODUCT_On_Customer_Query.Click_On_Customer_Query(Select_PRODUCT_On_Customer_Query.java:114)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Click_On_Customer_Query -->
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Click_On_Select_Product" time="1.853"/>
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Click_On_Query_History" time="6.31"/>
  <testcase classname="SIP.Select_PRODUCT_On_Customer_Query" name="Match_Product_Query_will_punched" time="0.51"/>
</testsuite> <!-- Default test -->
